plugins/modules/system/gconftool2_info.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xfconf_info
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Retrieve XFCE4 configurations
```

Comment:
I think some docs fields need to be updated :)

Reply:
Yup, that was a shameless copy-paste. Fixed now.